packages/xrpl/test/models/NFTokenMint.test.ts

```
@@ -1,11 +1,12 @@
-import { assert } from 'chai'
+import { stringToHex } from '@xrplf/isomorphic/src/utils'
 
-import {
-  convertStringToHex,
-  validate,
-  ValidationError,
-  NFTokenMintFlags,
-} from '../../src'
+import { NFTokenMintFlags } from '../../src'
+import { validateNFTokenMint } from '../../src/models/transactions/nftokenMint'
```

ESLint

[error] Missing file extension for "../../src/models/transactions/nftokenMint"

(import/extensions)